docs/_docs/1_1_spark_quick_start_guide.md

```diff
@@ -13,13 +13,17 @@ After each write operation we will also show how to read the data both snapshot
 
 ## Setup
 
-Hudi works with Spark-2.x & Spark 3.x versions. You can follow instructions [here](https://spark.apache.org/downloads.html) for setting up spark. 
+Hudi works with Spark-2.4.4+ & Spark 3.x versions. You can follow instructions [here](https://spark.apache.org/downloads.html) for setting up spark. 
```

Comment:
@n3nash : should we say 2.4.3? or 2.4.4 is fine?

Reply:
Let's put 2.4.3 since that's the right one